triplea_maps.yaml

```diff
@@ -317,6 +317,22 @@
     <br>http://tripleadev.1671093.n2.nabble.com/World-War-II-Global-1940-Original-tp7314662p7314662.html
     <br>
   version: 3.9
+- mapName: World War II Global Balanced
+  url: https://github.com/regularkyd/G40Balanced/blob/master/World%20War%20II%20G40%20Balanced.zip
```

Comment:
We need a link for a map that is hosted in triplea-maps, so it is under admin control. You could change this map whenever you want to, and players would download it. This could be a vector for introducing malicious content, hence the requirement that the map be in the triplea-maps repo.

Per the link/instructions I sent in the source forge thread, please submit a request for the map zip to be incorporated to triple-maps here: https://github.com/triplea-maps/Project/issues/new